src/guiguts/file.py

```diff
@@ -214,6 +217,14 @@ def interpret_bin(self, bin_dict: BinDict) -> None:
         Args:
             bin_dict: Dictionary loaded from bin file
         """
+        md5checksum = bin_dict.get(BINFILE_KEY_MD5CHECKSUM)
+        if md5checksum and md5checksum != self.get_md5_checksum():
+            logger.error(
+                """Main file and bin (.json) file do not match.
```

Comment:
It's really annoying that python isn't going to remove the line indents of this multiline string like php does with its HEREDOCs to allow us to indent this with the rest of the code easily. And it's cumbersome to use multiple strings to force the code layout (although it works, I think -- not sure what black will do with it though):
```python
logger.error(
    "Main file and bin (.json) file do not match.\n"
    "File may have been edited in a different editor.\n"
    "You may continue, but page boundary positions\n"
    "may not be accurate."
)

Reply:
Good news - black accepts that format/layout quite happily.